pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -46,6 +46,10 @@ const (
 	GetFSFreezeStatus AgentCommand = "guest-fsfreeze-status"
 
 	pollInitialInterval = 10 * time.Second
+
+	// verbosity levels
+	v3 = 3
```

Comment:
Redundant

Reply:
linter gives error otherwise that magic number is used